src/ApiManagement/ApiManagement/help/Get-AzApiManagementProduct.md

```diff
@@ -54,6 +60,21 @@ This command get an API Management product by ID.
 
 ## PARAMETERS
 
+### -ApiId
+ApiId of the Api to find the correlated products. This parameter is optional.
+
+```yaml
+Type: System.String
+Parameter Sets: GetByApiId
+Aliases:
+
+Required: False
```

Comment:
@solankisamir the help for this file will need to be regenerated when this parameter is made mandatory